<?xml version="1.0"?>
<launch>

    <arg name="config_file_path" default="$(find depthai_ros)/config/config.json"/>
    <arg name="depth_map_topic" default="/depth_map"/>
    <arg name="landmark_topic" default="/landmark"/>
    <arg name="disparity_threshold" default="200"/>
    <arg name="rate" default="10"/>

    <node pkg="depthai_ros" name="depth_map_publisher_node" type="depth_map_publisher_node" output="screen">
        <param name="config_file_path" value="$(arg config_file_path)"/>
        <param name="depth_map_topic" value="$(arg depth_map_topic)"/>
        <param name="landmark_topic" value="$(arg landmark_topic)"/>
        <param name="disparity_threshold" value="$(arg disparity_threshold)"/>
        <param name="rate" value="$(arg rate)"/>
    </node>

</launch>
